src/Prefixer.js

```diff
@@ -59,6 +59,13 @@ export function prefix (value, length) {
 		// justify-content
 		case 4968:
 			return replace(replace(value, /(.+:)(flex-)?(.*)/, WEBKIT + 'box-pack:$3' + MS + 'flex-pack:$3'), /s.+-b[^;]+/, 'justify') + WEBKIT + value + value
+		// justify-self
+		case 4200:
+			if (!match(value, /flex-|baseline/)) return MS + 'grid-column-align' + substr(value, length) + value
+			break
```

Comment:
@thysultan 

I re-checked the implementation of `autoprefixer`, it only excludes `flex-|baseline` from being prefixed with grid:

https://github.com/postcss/autoprefixer/blob/3041c7bfced438875d668767432666b10a88ef36/lib/hacks/grid-column-align.js#L7-L9
https://github.com/postcss/autoprefixer/blob/3041c7bfced438875d668767432666b10a88ef36/lib/hacks/grid-row-align.js#L7-L9

This allows us to simplify the regexp even further, resulting in an even smaller size.